resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -675,6 +676,20 @@ private[spark] object Config extends Logging {
       .checkValue(value => value > 0, "Maximum number of pending pods should be a positive integer")
       .createWithDefault(Int.MaxValue)
 
+  val KUBERNETES_JOB_MIN_MEMORY = ConfigBuilder("spark.kubernetes.job.min.memory")
```

Comment:
Since this new property is Volcano specific - should it be namespaced like `spark.kubernetes.volcano.job.min.memory` ?
Or is it better to use the current name and be (re-)used in the future by something else, e.g. YuniKorn ?

Reply:
We have some discussion on this with Yunikorn and Volcano team, we finally think `spark.kubernetes.job` is better than  `spark.kubernetes.xxx.job`:
- There are many other common arguments like queue, priorityClass, for this general parameter, we can extract to avoid 
redundant of configuration.
- Other customized scheduler can also use this configuration directly.